framework/src/main/java/dev/ikm/komet/framework/panel/axiom/ClauseView.java

```diff
@@ -420,10 +447,10 @@ private void setupForFeature() {
         openConceptButton.setOnMouseClicked(this::handleShowFeatureNodeClick);
         StringBuilder builder = new StringBuilder();
         builder.append("⒡ ");
-        Optional<Concept> optionalTypeConcept = this.axiomVertex.propertyAsConcept(TinkarTerm.FEATURE_TYPE);
-        Optional<Concept> optionalConcreteDomainOperator = this.axiomVertex.propertyAsConcept(TinkarTerm.CONCRETE_DOMAIN_OPERATOR);
-        ConceptFacade typeConcept = (ConceptFacade) optionalTypeConcept.get();
-        ConceptFacade concreteDomainOperatorConcept = (ConceptFacade) optionalConcreteDomainOperator.get();
+        Optional<ConceptFacade> optionalTypeConcept = this.axiomVertex.propertyAsConcept(TinkarTerm.FEATURE_TYPE);
+        Optional<ConceptFacade> optionalConcreteDomainOperator = this.axiomVertex.propertyAsConcept(TinkarTerm.CONCRETE_DOMAIN_OPERATOR);
+        ConceptFacade typeConcept = optionalTypeConcept.get();
+        ConceptFacade concreteDomainOperatorConcept = optionalConcreteDomainOperator.get();
```

Comment:
Can these two line (`typeConcept` & `concreteDomainOperatorConcept`) be moved in the if-condition?
Reasoning: `.get()` could raise a runtime exception if not present.